### PR TITLE
Add Non-TCK related FATS for MP Metrics 5.1

### PR DESCRIPTION
**Pull request by Channyboy**
- Remove beta flag from existing FATS
- Add io.openliberty.microprofile.metrics.internal.5.1_fat_tck with new FATS for optional MP Config for MP Metrics 5.1
- Add feature replacement for mpMetrics-5.1 for io.openliberty.microprofile.metrics.internal.5.x.monitor_fat

#build

### CODE REVIEW

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/resources/logging.properties
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/fat/resources/logging.properties:1]: Is this file needed to be checked in?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/DefaultBucketServer/jvm.options [hunk 1]
```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/DefaultBucketServer/jvm.options]: Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/MicrometerPrometheus/jvm.options [hunk 2]
```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/MicrometerPrometheus/jvm.options:1]: Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/MicrometerUseless/jvm.options [hunk 3]
```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/MicrometerUseless/jvm.options:1]: Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/NoMicrometerCore/jvm.options [hunk 4]
```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/NoMicrometerCore/jvm.options:1]: Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?

File: dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/NonExistentLibrary/jvm.options [hunk 5]
```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.metrics.internal.5.1_fat/publish/servers/NonExistentLibrary/jvm.options:1]: Since, mpMetrics-5.1 is still in beta, shouldn't this file set the BETA JVM prop to be true, for the FATs to run properly?

**Review by pgunapal — DISMISSED** [review]
Looks good, minor comments.

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#libby

**Review by pgunapal — DISMISSED** [review]
LGTM

**Comment by Channyboy** [comment]
#pbbuild

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build
